server/plugin/api.go

```diff
@@ -489,14 +489,21 @@ func (p *Plugin) getReviewersListString(reviewersList []serializers.Reviewer) st
 }
 
 func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.Request) {
+	webhookSecret := r.URL.Query().Get(constants.AzureDevopsQueryParamWebhookSecret)
+	if webhookSecret == "" || webhookSecret != p.getConfiguration().WebhookSecret {
```

Comment:
can we use this here https://github.com/Brightscout/mattermost-plugin-servicenow/blob/e9d9ae34f20bc52872e7ab022924b582208b4ea1/server/plugin/api.go#L824

Reply:
have to change some logic as here we are receiving as query param and not URL encoded FORM